api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -2005,6 +2004,17 @@ default boolean hasStyling() {
     return !this.style().isEmpty();
   }
 
+  /**
+   * Tests if this component is empty.
```

Comment:
This is kinda misleading, is the component `text("", NamedTextColor.RED)` empty? What about a translatable component that renders to the empty component?